src/gitops/components/appset/EventsTab.scss

```diff
@@ -0,0 +1,10 @@
+.application-set-details-page {
```

Comment:
Similarly, could we delete this file and use the shared one instead, in

/shared/EventsTab/EventsTab

?